problemsets-mat223/linearalbegra.tex

```diff
@@ -3091,32 +3091,79 @@ \section*{Task 2.3: Pat and Jamie}
 	\end{parts}
 
 	\begin{definition}[Range]
-		The \emph{range} (or \emph{image}) of a linear transformation $T:V\to W$ is the set of vectors that
-		$T$ can output.  That is,
+		The \emph{range} (or \emph{image}) of a linear transformation $T:V\to W$ 
+		is the set of vectors that $T$ can output. That is,
 		\[
-			\Range(T)=\{\vec y\in W:\vec y=T\vec x\text{ for some }\vec x\in V\}.
+			\Range(T)=\Set{\vec y\in W \given \vec y=T\vec x\text{ for some }\vec x\in V}.
 		\]
 	\end{definition}
 	\begin{definition}[Null Space]
-		The \emph{null space} (or \emph{kernel}) of a linear transformation $T:V\to W$ is the
-		set of vectors that get mapped to zero under $T$.  That is,
+		The \emph{null space} (or \emph{kernel}) of a linear transformation $T:V\to W$
+		is the set of vectors that get mapped to zero under $T$. That is,
 		\[
-			\Null(T)=\{\vec x\in V:T\vec x=\vec 0\}.
+			\Null(T)=\Set{\vec x\in V \given T\vec x=\vec 0}.
 		\]
 	\end{definition}
 
 	\question
-	Let $\mathcal P:\R^2\to\R^2$ be projection onto $\Span\{\vec u\}$ where $\vec u=\mat{2\\3}$ (like before).
+	Let $\mathcal P:\R^2\to\R^2$ be projection onto $\Span\Set{\vec u}$ where
+	$\vec u=\mat{2\\3}$ (like before).
 	\begin{parts}
 		\item What is the range of $\mathcal P$?
+			\begin{solution}
+				$\Range(\mathcal P)=\Span\Set{\vec u}$.
+
+				$\mathcal P(\vec x)$ is by definition the vector in $\Span\Set{\vec u}$ 
+				that is closest to $\vec x$, so in particular 
+				$\mathcal P(\vec x) \in \Span\Set{\vec u}$ for all $\vec x\in\R^2$.
+				Therefore $\Range(\mathcal P)\subseteq\Span\Set{\vec u}$. 
+				
+				On the other hand, $\mathcal P(\alpha \vec u)=\alpha\mathcal P(\vec u)=\alpha\vec u$ 
+				for any scalar $\alpha$, and so $\Range(\mathcal P)\supseteq\Span\Set{\vec u}$
```

Comment:
`\subseteq` to `=`

Reply:
Making that change, but my intention here was for each line to show one direction of set containment. I could then either leave the students to conclude that containment both ways implies equality, or write it out on the next line.